netbout-web/src/main/java/com/netbout/rest/bout/TkAttach.java

```diff
@@ -156,6 +176,31 @@ private String name(final Request file) throws IOException {
     }
 
     /**
+     * Copy the multipart data to a file and validate it.
+     * @param src Multipart request
+     * @param dst File
+     * @param name Filename
+     * @throws IOException If copy or validate fails
+     */
+    private void copyAndValidate(final Request src, final File dst,
+        final String name) throws IOException {
+        this.copy(src, dst);
+        this.validate(dst, name);
+    }
+
+    /**
+     * Copy data from the a multipart request to a file.
+     * @param src Multipart Request
+     * @param dst File
+     * @throws IOException If fails
+     */
+    private void copy(final Request src, final File dst) throws IOException {
```

Comment:
@jhyle instead of creating this method you can reuse `Files.copy(src.body, dst.toPath())`